wpilibc/src/main/native/include/frc/kinematics/SwerveDriveKinematics.inc

```diff
@@ -81,6 +81,28 @@ ChassisSpeeds SwerveDriveKinematics<NumModules>::ToChassisSpeeds(
           units::radians_per_second_t{chassisSpeedsVector(2)}};
 }
 
+template <size_t NumModules>
+template <typename... ModuleStates>
+ChassisSpeeds SwerveDriveKinematics<NumModules>::ToChassisSpeeds(
```

Comment:
Is there a way to perhaps make one method call the other to avoid duplicate code?

Reply:
I think it would bring unnecessary complexity, if it is possible then RamseteCommand could be changed too.

Reply:
This has nothing to do with RamseteCommand. You are adding a new overload to SwerveDriveKinematics with duplicate code.

Reply:
Oops, sorry complete misread the file. Yeah one can all the other just forgot to do that. Thanks!